### ISSUE TITLE
BTC / Stablesats explainer

### ISSUE THREAD

**Issue by designsats** [GaloyMoney/galoy-mobile#2876]
Problem
Too much text explaining the difference between Stablesats and BTC

Solution
Graphical representation and more compact text: 

EN: Use your Stablesats account in Blink to keep the money in your wallet stable in fiat (dollar) terms. Use your Bitcoin account if you're stacking sats and are okay with your fiat balance changing all the time.

ES: Utiliza tu cuenta Stablesats en Blink para mantener el dinero de tu cartera estable en términos fiat (dólares). Utiliza tu cuenta Bitcoin si estás acumulando sats y te parece bien que tu saldo fiat cambie todo el tiempo.

SVG: 
![btc-usd-graphs](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/dfa899eb-fa29-417b-8efb-e4e0287d4347)

![Stablesats](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/934c6db0-f630-421a-966e-632bcb908c6c)

**Comment by nicolasburtey** [comment]
while I think that make sense to have stablesats going down if we're tweeting about it - ie: to reflect the inflation, I don't think it will help for the comprehension of stablesats in the first place.

basically, only if you're very advanced in the comprehension of money and finance you get the graphical representation as it is now. 

if you're just getting started, having stablesats as a straight line would help in the comprehension imo

**Comment by designsats** [comment]
You are right. I hate it, but you are right. My maxi-bias is showing. Can we settle on tiny tiny down angle? :) 

![Stablesats](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/16e794de-4c19-44c8-bc99-77818c4b8c44)

**Comment by designsats** [comment]
![btc-stables-graphs](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/509b3a19-d6bf-4a1c-985f-fadf5c39deef)

**Comment by designsats** [comment]
@nicolasburtey is it ok like this? (image above)
